x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/rollup/job/IndexerUtilsTests.java

```diff
@@ -361,14 +362,14 @@ public void testKeyOrderingLong() {
             List<CompositeAggregation.Bucket> foos = new ArrayList<>();
 
             CompositeAggregation.Bucket bucket = mock(CompositeAggregation.Bucket.class);
-            LinkedHashMap<String, Object> keys = new LinkedHashMap<>(3);
+            Map<String, Object> keys = Maps.newLinkedHashMapWithExpectedSize(3);
             keys.put("foo.date_histogram", 123L);
 
             char[] charArray = new char[IndexWriter.MAX_TERM_LENGTH];
             Arrays.fill(charArray, 'a');
             keys.put("bar.terms", new String(charArray));
             keys.put("abc.histogram", 1.9);
-            keys = shuffleMap(keys, Collections.emptySet());
+            keys = shuffleMap((LinkedHashMap<String, Object>) keys, Collections.emptySet());
```

Comment:
```suggestion
            keys = shuffleMap(keys, Collections.emptySet());
```

Looks like this was accidentally missed